torch/utils/data/datapipes/iter/selecting.py

```diff
@@ -1,22 +1,14 @@
-import warnings
 from typing import Callable, Iterator, TypeVar
 
 from torch.utils.data import IterDataPipe, functional_datapipe
 from torch.utils.data.datapipes.dataframe import dataframe_wrapper as df_wrapper
+from torch.utils.data.datapipes.utils.common import DILL_AVAILABLE, check_lambda_fn
```

Comment:
Why `DILL_AVAILABLE` is needed here?

Reply:
Actually they aren't so I am going to remove them now.